layer.go

```diff
@@ -112,6 +112,13 @@ func (l *Layer) Init(ctx context.Context, desc *LayerDescription, r io.ReaderAt)
 			return fmt.Errorf("claircore: layer %v: unable to create fs.FS: %w", desc.Digest, err)
 		}
 		l.sys = sys
+	case `application/x.filesystem+uncompressed`:
```

Comment:
What is the reasoning behind this content type? (Maybe `application/x-filesystem`) is the uncompressed implied through omission?

Reply:
Piotr and I weren't able to find a good candidate from existing types, so we came up with this one.
We can absolutely switch that to `application/x-filesystem`, no worries.